src/plone/restapi/serializer/atfields.py

```diff
@@ -60,10 +61,11 @@ def __call__(self):
 class TextFieldSerializer(DefaultFieldSerializer):
 
     def __call__(self):
+        mt = getToolByName(self.context, 'mimetypes_registry')
```

Comment:
@erral just a minor issue. I prefer to use variable names that allow the developer to understand what's in there. I would recommend to use "mimetypes_registry" instead of "mt".